numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -57,12 +59,7 @@
 from numba.cuda.cudadrv import enums, drvapi, nvrtc
 
 from cuda.bindings import driver as binding
-from cuda.core.experimental import (
-    Linker,
-    LinkerOptions,
-    ObjectCode,
-)
-
+from cuda.core.experimental import Linker, LinkerOptions, ObjectCode
 from cuda.bindings.utils import get_cuda_native_handle
 from cuda.core.experimental import (
     Stream as ExperimentalStream,
```

Comment:
Not sure if already too late, but the name `ExperimentalStream` sounds a bit non-intuitive from the first look. For context because there are separate stream object in cuda core and Numba-CUDA, I too also have to sometime rename `cuda.core.experimental.Stream` to something like `CudaPythonStream` or use the qualified name.